src/entities/applet/ui/ActivityAnalyticsCard.tsx

```diff
@@ -1,13 +1,13 @@
 import { FC } from 'react';
 
 import { styled } from '@tamagui/core';
-import { useTranslation } from 'react-i18next';
 
-import { Activity } from '@entities/applet';
-import { Center, Text } from '@shared/ui';
+import { AnalyticsChart } from '@app/features/analytics-chart';
+import { ActivityResponsesDto } from '@shared/api';
+import { Box, Center, Text } from '@shared/ui';
 
 type Props = {
-  activity: Activity;
+  responseData: ActivityResponsesDto;
```

Comment:
The component props should not rely on DTO types.
It should only accept the data it needs and the data should not be zipped into one object.
The better way: create two properties: name, responses